packages/eudev.rb

```diff
@@ -28,16 +26,15 @@ class Eudev < Package
 
   def self.build
     system 'sed -i s,/usr/bin/xsltproc,xsltproc,g man/make.sh'
-    system './autogen.sh'
-    system './configure',
-           '--enable-hwdb',
-           '--enable-rule-generator',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--with-rootprefix=#{CREW_PREFIX}",
-           "--with-rootlibdir=#{CREW_LIB_PREFIX}",
-           "--with-rootrundir=#{CREW_PREFIX}/run",
-           "--with-rootlibexecdir=#{CREW_PREFIX}/libexec"
+    system 'NOCONFIGURE=1 ./autogen.sh'
+    system "#{CREW_ENV_OPTIONS} ./configure \
+              #{CREW_OPTIONS} \
+              --enable-hwdb \
+              --enable-rule-generator \
+              --with-rootprefix=#{CREW_PREFIX} \
+              --with-rootlibdir=#{CREW_LIB_PREFIX} \
+              --with-rootrundir=/run \
+              --with-rootlibexecdir=#{CREW_PREFIX}/libexec"
```

Comment:
Don't all these options default to the correct locations already?

Reply:
You are right.